src/hotspot/cpu/x86/stubGenerator_x86_64_aes.cpp

```diff
@@ -2615,29 +2616,36 @@ void StubGenerator::aesctr_encrypt(Register src_addr, Register dest_addr, Regist
   // Save encrypted counter value in xmm0 for next invocation, before XOR operation
   __ movdqu(Address(saved_encCounter_start, 0), xmm0);
   // XOR encryted block cipher in xmm0 with PT to produce CT
-  __ evpxorq(xmm0, xmm0, Address(src_addr, pos, Address::times_1, 0), Assembler::AVX_128bit);
```

Comment:
This could be fixed alternatively by using mask register with evpxorq. That will have lower impact on performance. @smita-kamath can share the changes needed.